freshmaker/lightblue.py

```diff
@@ -42,6 +42,28 @@
 import koji
 
 
+class ImageGroup:
+    def __init__(self, image, lightblue):
+        parsed_nvr = koji.parse_NVR(image.nvr)
+        repositories = image.get_registry_repositories(lightblue)
+        self.name = parsed_nvr["name"]
+        self.version = parsed_nvr["version"]
+        self.repos = {x["repository"] for x in repositories}
+
+    def __eq__(self, other):
+        return all(
+            [self.name == other.name and self.version == other.version and self.repos == other.repos]
+        )
```

Comment:
It'd be simpler and more efficient to change this to:
```python
return self.name == other.name and self.version == other.version and self.repos == other.repos
```

As of now, you are create a list with a single boolean value and then calling the `all` function against it.

Reply:
ahh i see, you're right. I'll make the changes and submit a PR. A quick question before that though: for lines of code that are long (exceeding 100 characters) is there a style guide that should be followed to break it into multiple lines? using backslashes, auto-black styler, etc?

**The code in question is below**
```
    def issubset(self, other):
        return self.name == other.name and self.version == other.version and self.repos.issubset(other.repos)
```

**What auto-black styler on VS code formats it to**
```
    def issubset(self, other):
        return (
            self.name == other.name
            and self.version == other.version
            and self.repos.issubset(other.repos)
        )
```

Reply:
@ronnyhlim I would go with what `black` would do. That tool is becoming a common auto-formatter used by many Python projects, so it's always safe to do as it suggests.